packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -117,7 +115,7 @@ class RenderParagraph extends RenderBox
     }
   }
 
-  static final String _placeholderCharacter = String.fromCharCode(PlaceholderSpan.placeholderCodeUnit);
+  static final String _placeholderCharacter = String.fromCharCode(Unicode.ORC);
```

Comment:
Does it matter that [RenderParagraph] uses the same placeholder character as [PlaceholderSpan]?

Reply:
Good point there. The code later does:
```
      int end = plainText.indexOf(_placeholderCharacter, start);
```
I'll restore `PlaceholderSpan.placeholderCodeUnit` and use a `Unicode.?` constant for it.